pkg/qemu/qemu.go

```diff
@@ -927,7 +937,11 @@ func Cmdline(ctx context.Context, cfg Config) (exe string, args []string, err er
 	const serialvChardev = "char-serial-virtio"
 	args = append(args, "-chardev", fmt.Sprintf("socket,id=%s,path=%s,server=on,wait=off,logfile=%s", serialvChardev, serialvSock, serialvLog))
 	// max_ports=1 is required for https://github.com/lima-vm/lima/issues/1689 https://github.com/lima-vm/lima/issues/1691
-	args = append(args, "-device", "virtio-serial-pci,id=virtio-serial0,max_ports=1")
+	serialvMaxPorts := 1
+	if *y.Arch == limayaml.S390X {
+		serialvMaxPorts++ // needed to avoid `virtio-serial-bus: Out-of-range port id specified, max. allowed: 0`
```

Comment:
This is still confusing to me. Why does it report "max. allowed: 0" when you pass `max_ports=1`?

Reply:
Might be a bug of qemu? idk